web_app/src/Register.js

```diff
@@ -40,6 +40,7 @@ class Register extends React.Component {
                 email: user.email,
                 photoURL: user.photoURL,
                 platform: "Email",
+                articles_raised_eyebrow: [],
```

Comment:
this will just do it for signup with email/password, not with other kinds of login